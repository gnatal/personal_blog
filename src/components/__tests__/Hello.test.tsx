import React from 'react';
import renderer from 'react-test-renderer';
import HelloYou from '../Hello';

test('It should craete a snapshot', () => {
  const component = renderer.create(<HelloYou name="pedro" />);
  const tree = component.toJSON();
  expect(tree).toMatchSnapshot();
});
