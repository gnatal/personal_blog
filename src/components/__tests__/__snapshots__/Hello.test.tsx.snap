// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`It should craete a snapshot 1`] = `
<p>
  pedro
</p>
`;
